test/tools/libtesteth/TestHelper.cpp

```diff
@@ -382,6 +382,35 @@ string compileLLL(string const& _code)
 #endif
 }
 
+#include <wasm-binary.h>
+#include <wasm-s-parser.h>
+
+/* from https://github.com/ewasm/evm2wasm */
+bytes wast2wasm(string input, bool debug = false) {
+  wasm::Module wasm;
+
+  try {
+    wasm::SExpressionParser parser(const_cast<char*>(input.c_str()));
+    wasm::Element& root = *parser.root;
+    wasm::SExpressionWasmBuilder builder(wasm, *root[0]);
+  } catch (wasm::ParseException& p) {
+    ostringstream err;
+    p.dump(err);
+    BOOST_ERROR("Error parsing wasm: " + err.str());
+  }
+
+  // FIXME: perhaps call validate() here?
+
+  wasm::BufferWithRandomAccess buffer(debug);
+  wasm::WasmBinaryWriter writer(&wasm, buffer, debug);
+  writer.write();
+
+  ostringstream output;
+  buffer.writeTo(output);
+
+  return bytes(output.begin(), output.end());
+}
+
 void checkHexHasEvenLength(string const& _str)
```

Comment:
do not change importCode function. 
the changes must be done to replaceLLL function 
but after this PR got merged please https://github.com/ethereum/cpp-ethereum/pull/4735/files

just rename replaceLLL function to compileSrcCode or something
and parse wasm and LLL from there

Reply:
Has been updated now.